### DISCUSSION TITLE
希望能在官方文档中展示社区插件、主题~

### DISCUSSION THREAD

**Discussion by Yue-plus** [vuepress/core#406] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/33195150/130853773-2a3f9a2a-825a-4ea1-8e2b-aeb6be9dbb02.png)

----------

我做的 VuePress 主题~
[GitHub 仓库](https://github.com/Yue-plus/vuepress-theme-oxygen-not-included) | [主题预览](https://oxygennotincluded.theme.yueplus.ink/)

**Comment by Mister-Hope** [comment, marked as answer]
Track #735
